tests/baselines/reference/superCallFromClassThatDerivesFromGenericTypeButWithNoTypeArguments1.errors.txt

```diff
@@ -11,6 +14,12 @@ tests/cases/compiler/superCallFromClassThatDerivesFromGenericTypeButWithNoTypeAr
     
     class B extends A {
                     ~
-!!! error TS2314: Generic type 'A<T1, T2>' requires 2 type argument(s).
+!!! error TS2707: Type parameter 'T1' implicitly has type 'any' because it was not supplied.
+                    ~
+!!! error TS2707: Type parameter 'T2' implicitly has type 'any' because it was not supplied.
         constructor() { super(value => String(value)); }
+                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2346: Supplied parameters do not match any signature of call target.

```

Comment:
This seems wrong. we already set `T1` and `T2` to any, so this should resolve with no errors, the `noImplicitAny` on the extends clause should be sufficient.